test/gerkin/tests/features/data_delete.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Remove record from the database.
+  Request endpoint: DELETE /data/{code}/{version}/delete
```

Comment:
Please change `code` to `registryname` and version to `versionnumber` to match openAPI spec. Also URI ends with /{ID}/delete pointing which record is to be deleted.